code/js/lib/jquery.i18n/jquery.i18n.parser.js

```diff
@@ -0,0 +1,309 @@
+/*!
```

Comment:
You see formatting of this file? It is terrible. You minify all external files. I believe that atom is able to show it normally when you need it to show what external function do.

Reply:
all external library files have been minified